src/test/java/com/atlassian/bitbucket/jenkins/internal/client/BitbucketClientFactoryImplTest.java

```diff
@@ -231,8 +231,8 @@ public void testGetWebHookCapabilities() {
 
     private BitbucketClientFactoryImpl getClientFactory(
             String url, BitbucketCredentials credentials) {
-        HttpRequestExecutor executor = new HttpRequestExecutorImpl(mockExecutor);
-        return new BitbucketClientFactoryImpl(url, credentials, objectMapper, executor);
+        HttpRequestExecutor executor = new HttpRequestExecutorImpl(mockExecutor, objectMapper);
+        return new BitbucketClientFactoryImpl(url, credentials, executor);
     }
 
```

Comment:
Would a unit test that makes sure that the client works for mirror requests be in order? We have tests for all other clients, and I think it is valuable, and also to have the response object (by way of a json file) in the repo is also helpful for future maintainers and extenders :)

Reply:
Oops, missed it